lib/portlayer/network/network.go

```diff
@@ -20,6 +20,8 @@ import (
 
 	log "github.com/Sirupsen/logrus"
 
+	"golang.org/x/net/context"
```

Comment:
hmm nope, this was the result of goimports but while at it I'll move them to context